File: LV1/1.2/quadrant/quadrant.cpp
#include <cstdio>

int quadrant(int x, int y)
{
  if (x > 0 && y > 0)
    return 1;
  if (x < 0 && y > 0)
    return 2;
  if (x < 0 && y < 0)
    return 3;
  return 4;
}

int main()
{
  int x, y;
  scanf("%d %d", &x, &y);
  printf("%d\n", quadrant(x, y));
}

File: LV1/1.2/quadrant/quadrant.go
package main

import "fmt"

func quadrant(x int, y int) int {
	if x > 0 && y > 0 {
		return 1
	} else if x < 0 && y > 0 {
		return 2
	} else if x < 0 && y < 0 {
		return 3
	} else {
		return 4
	}
}

func main() {
	var x, y int

	fmt.Scanf("%d\n%d", &x, &y)
	fmt.Println(quadrant(x, y))
}
